tsconfig.base.json

```diff
@@ -6,15 +6,26 @@
     "module": "ESNext",
     "moduleResolution": "Node",
     "noEmit": true,
+    "noImplicitThis": true,
     "resolveJsonModule": true,
     "skipLibCheck": true,
     "strict": false,
     "strictBindCallApply": true,
     "strictFunctionTypes": true,
     "target": "ESNext",
-    "types": ["jest", "jest-axe", "jest-puppeteer", "node"],
+    "types": [],
     "paths": {
-      "govuk-frontend": ["./packages/govuk-frontend/src/govuk/all.mjs"]
+      "govuk-frontend": ["./packages/govuk-frontend/src/govuk/all.mjs"],
+      "govuk-frontend-stats": ["./shared/stats/src/index.mjs"],
+      "govuk-frontend-config": ["./shared/config/index.js"],
+      "govuk-frontend-helpers": ["./shared/helpers/index.js"],
+      "govuk-frontend-helpers/*": ["./shared/helpers/*.js"],
+      "govuk-frontend-helpers/jest/*": ["./shared/helpers/jest/*"],
+      "govuk-frontend-lib": ["./shared/lib/index.js"],
+      "govuk-frontend-lib/*": ["./shared/lib/*.js"]
     }
+  },
+  "typeAcquisition": {
+    "enable": false
```

Comment:
Added one extra flag

This prevents editors from automatically downloading `@types/*` packages to speed up first startup time. We already declare all the types we'd like anyway

It's highlighted that a few packages are missing, but other than `@types/js-yaml` they're out of date and cause issues with our code accessing things that aren't typed

```console
@types/js-yaml
@types/js-beautify
@types/shuffle-seed
```